README.md

```diff
@@ -259,6 +259,95 @@ SECRET_KEY_SEED=z1Ajrg3wpDYiKS1EKAZKDd8qwR5mG3wKvh6gYi5YEz9T89P ./did key create
 }
 ```
 
+### Anchored Resources
+
+Parameters:
+- Input: Either pass in a `--file` parameter, pipe something in via `stdin`,
+  or provide a `--url` parameter (which will be fetched).
+- `--file, -f` - Optional input file containing the resource to be anchored/hashed.
+- `--canonicalize` - Optional `canonicalizationAlgorithm` value. The resource
+  to be anchored will first be canonicalized with this algorithm. For JSON or
+  JSON-LD type resources, the default algorithm is `jcs`.
+- `url` - Optional URL parameter for the anchored resource. If no input `--file`
+  or `stin` value is specified, the CLI will attempt to retrieve the resource
+  at the `url`.
+- Id: If a `url` param is provided, it will be used as the `anchoredResource.id`.
+  Otherwise, the CLI will use the `id` property of the input resource.
+  If neither exists (or if the resource is not a JSON object), no id will be
+  used.
+
+#### Creating an `anchoredResource`
+
+To create an anchored resource (which includes a content hash and
+a canonicalization algorithm): 
+
+```
+./did anchor create -f example-vc.json --canonicalize jcs
+{
+  "anchoredResource": {
+    "id": "<id of the object in example-vc.json>",
+    "contentHash": {
+      "digestValue": "z1Aaj5A4UCsdMpXwdYAReXa4bxWYiKJtdAvB1zMzCHtCbtD",
```

Comment:
We may want this to be `digestMultibase` instead -- so we can have a new term that is CBOR-LD compressible.

Reply:
@dlongley  - sure, will change to `digestMultibase`

Reply:
Renamed.